types/xo/index.d.ts

```diff
@@ -0,0 +1,49 @@
+// Type definitions for xo 0.26
+// Project: https://github.com/xojs/xo#readme
+// Definitions by: Piotr Błażejewicz (Peter Blazejewicz) <https://github.com/peterblazejewicz>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import eslint = require('eslint');
+
+export {};
+/**
+ * From T pick a set of properties K
+ */
+type Pick<T, K extends keyof T> = {
+    [P in K]: T[P];
+};
+
+export type getErrorResults = typeof eslint.CLIEngine.getErrorResults;
+export type outputFixes = typeof eslint.CLIEngine.outputFixes;
```

Comment:
I don't think this is a type that's exported.
```suggestion
export const outputFixes : typeof eslint.CLIEngine.outputFixes;
```

Reply:
Yes, very good catch, I've overlooked these two as beign import from ESLint, sorry. 
The definition is rewritten to match shapes and I've added sample usage tests too.
Thanks!